Python/ceval.c

```diff
@@ -143,7 +143,7 @@ lltrace_instruction(_PyInterpreterFrame *frame,
     const char *opname = _PyOpcode_OpName[opcode];
     assert(opname != NULL);
     int offset = (int)(next_instr - _PyCode_CODE(frame->f_code));
-    if (HAS_ARG(_PyOpcode_Deopt[opcode])) {
+    if (HAVE_ARGUMENT <= _PyOpcode_Deopt[opcode]) {
```

Comment:
HAVE_ARGUMENT isn't used directly anywhere else. I think casting and using HAS_ARG might be better.

Reply:
But we don't have to worry about things like pseudo-opcodes, right? This seems like exactly the sort of thing `HAVE_ARGUMENT` is meant for...

Reply:
Kind of. HAVE_ARGUMENT was there before we had pseudo-opcodes, and HAS_ARG just compared with it. Now we have pseudo-opcodes, and if I saw HAVE_ARGUMENT in the code now my first thought would be that this is a bug (a place where we didn't update for pseudo-opcodes).  So the trick is to make this not look like a bug.